clients/cobol-lsp-vscode-extension/src/extension.ts

```diff
@@ -13,25 +13,24 @@
  */
 
 import * as vscode from "vscode";
-import * as path from "path";
+
 import { changeDefaultZoweProfile } from "./commands/ChangeDefaultZoweProfile";
 import { editDatasetPaths } from "./commands/EditDatasetPaths";
 import { fetchCopybookCommand } from "./commands/FetchCopybookCommand";
-import { C4Z_FOLDER, DEPENDENCIES_FOLDER, GITIGNORE_FILE, REASON_MSG } from "./constants";
+import { C4Z_FOLDER, GITIGNORE_FILE} from "./constants";
 import { LANGUAGE_ID, SETTINGS_SECTION } from "./constants";
 import { CopybookFix } from "./services/CopybookFix";
 import { CopybooksCodeActionProvider } from "./services/CopybooksCodeActionProvider";
 import { CopybooksDownloader } from "./services/CopybooksDownloader";
 import { CopybooksPathGenerator } from "./services/CopybooksPathGenerator";
 import { initializeSettings, createFileWithGivenPath } from "./services/Settings";
 
-import {resolveLocalCopybooksCommand} from "./commands/ResolveLocalCopybooksCommand";
-import { LanguageClientService } from "./services/LanguageClientService";
-import { PathsService } from "./services/PathsService";
-import { ProfileService } from "./services/ProfileService";
-import {CopybookResolver} from "./services/settings/CopybookResolver";
-import {LocalCopybookResolver} from "./services/settings/LocalCopybookResolver";
-import { ZoweApi } from "./services/ZoweApi";
+import {CopybookURI} from "./services/CopybookURI";
+import {LanguageClientService} from "./services/LanguageClientService";
```

Comment:
would be better if we have same formatting

Reply:
Looks like the new format works correctly, for example:
```
import {ZoweApi} from "./services/ZoweApi";
```
is better than the prev version:
```
import { ZoweApi } from "./services/ZoweApi";
```